.travis.yml

```
@@ -0,0 +1,19 @@
+# once a new release tag is recreated, circle.yml sends a request to travis.yml
+# travis.yml builds the documentation in this repo and create a pull request
+language: node_js
+notifications:
+  email: false
+before_install:
+  - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" > .npmrc
+
+jobs:
+  include:
+    - name: "Build Maps Release Docs"
+      os: osx
+      osx_image: xcode10.2
+      if: env(SDK_FLAVOR) = maps AND env(RELEASE_TAG) IS PRESENT
+      script:
+        - npm link
+        - git config user.email "MapboxCI@users.noreply.github.com"
+        - ./scripts/docs-update-maps.sh $RELEASE_TAG
+        - docs-release-pull-request -s $SDK_FLAVOR -t $RELEASE_TAG -b publisher-staging -p ios -r mapbox-gl-native-ios
```

Before merge, we can remove the branch flag (default is publisher-production)

```suggestion
        - docs-release-pull-request -s $SDK_FLAVOR -t $RELEASE_TAG -p ios -r mapbox-gl-native-ios
```